.github/workflows/somehow-work.yaml

```
@@ -0,0 +1,68 @@
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.branch }}
+
+      - name: Install Nix
+        uses: DeterminateSystems/nix-installer-action@main
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          
+      - name: Set up Nix cache
+        uses: DeterminateSystems/magic-nix-cache-action@main
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)